index.qmd

```diff
@@ -41,6 +41,23 @@ install.packages(pkgs)
 remotes::install_github("tidymodels/modeldatatoo")
 ```
 
+It is also recommended that you run the following code. This ensures that the 
```

Comment:
this sentence is missing its second half 2️⃣

Reply:
fixed by simon!